plugin/storage/cassandra/spanstore/reader.go

```diff
@@ -414,25 +428,27 @@ func (s *SpanReader) executeQuery(span opentracing.Span, query cassandra.Query,
 	tableMetrics.Emit(err, time.Since(start))
 	if err != nil {
 		logErrorToSpan(span, err)
-		span.LogFields(otlog.String("query", query.String()))
+		span.SetAttributes(attribute.Key("query").String(query.String()))
 		s.logger.Error("Failed to exec query", zap.Error(err), zap.String("query", query.String()))
 		return nil, err
 	}
 	return retMe, nil
 }
 
-func startSpanForQuery(ctx context.Context, name, query string) (opentracing.Span, context.Context) {
-	span, ctx := opentracing.StartSpanFromContext(ctx, name)
-	ottag.DBStatement.Set(span, query)
-	ottag.DBType.Set(span, "cassandra")
-	ottag.Component.Set(span, "gocql")
-	return span, ctx
+func startSpanForQuery(ctx context.Context, name, query string, tp trace.Tracer) (context.Context, trace.Span) {
+	ctx, span := tp.Start(ctx, name)
```

Comment:
```suggestion
func (s *SpanReader) startSpanForQuery(ctx context.Context, name, query string) (context.Context, trace.Span) {
	ctx, span := s.tracer.Start(ctx, name)
```

Reply:
Ok, cool! now it's become a member function. Thanks for this direction.